data/json/monsterdrops/zombie_survivor.json

```diff
@@ -9,10 +9,47 @@
       { "distribution": [
         { "group": "book_survival", "prob": 25 },
         { "group": "gear_survival", "prob": 25, "damage": [ 1, 4 ] },
-        { "group": "guns_survival", "prob": 25, "damage": [ 1, 4 ] },
+        { "group": "survivor_weapons", "prob": 50, "damage": [ 1, 4 ] },
+        { "group": "survivor_grenades", "prob": 15, "damage": [ 1, 4 ] },
         { "group": "stash_food", "prob": 40 }
       ] },
       { "item": "cash_card" }
     ]
+  },
+  {
+    "id" : "survivor_melee",
+    "type" : "item_group",
+    "items": [
+      [ "ax", 5 ],
+      [ "bwirebat",15 ],
+      [ "crowbar", 5 ],
+      [ "nailboard", 15 ],
+      [ "machete", 5 ],
+      [ "makeshift_crowbar", 25 ],
+      [ "makeshift_knife", 20 ],
+      [ "makeshift_machete", 15 ],
+      [ "makeshift_halberd", 5 ],
+      [ "nailbat", 15 ],
+      [ "spear_knife", 15 ]
+    ]
+  },
+  {
+    "id" : "survivor_grenades",
+    "type" : "item_group",
+    "items": [
+      [ "gasbomb_makeshift", 30 ],
+      [ "matchbomb", 50 ],
+      [ "molotov", 100 ],
+      [ "nail_bomb", 100 ],
+      [ "pipebomb", 30 ]
+    ]
+  },
+  {
+    "id" : "survivor_weapons",
+    "type" : "item_group",
+    "items":[
+      { "group": "survivor_melee", "prob": 60 },
+      { "group": "guns_survival", "prob": 40 }
```

Comment:
It was copied from `cop_weapons` which had the same balance. If anything the inverse is preferable here as improvised guns are markedly underused. Part of the intention of my recent PR's has been to move these earlier in the game.